packages/server/src/data/schemas/post-schema.js

```diff
@@ -0,0 +1,25 @@
+const CommentSchema = require("./comment-schema");
+const { Schema } = require("mongoose");
+const {
+  Types: { ObjectId },
+} = Schema;
+
+const LocationSchema = new Schema({
+  type: String,
+  coordinates: [Number],
+});
+
+const PostSchema = new Schema({
+  title: String,
+  body: String,
+  date: Date,
+  numUpvotes: Number,
+  numDownvotes: Number,
+  location: LocationSchema,
+  comments: [CommentSchema],
+  userId: ObjectId,
+  username: String,
+  tag: String,
+});
```

Comment:
just checking... no more anonymous posting?

Reply:
If we want to do anonymous, I guess we can just make username/userId undefined. Basically anonymous posts don't count towards your rep.